DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Send/RootObjectSender.cs

```diff
@@ -37,10 +36,7 @@ public RootObjectSender(Func<Account, string, ITransport> transportFactory)
 
     onOperationProgressed?.Invoke("Uploading...", null);
 
-    // POC: FETCHING ACCOUNTS BY ID ONLY IS UNSAFE, we should filter by server first, but the server info is not stored on the ModelCard
```

Comment:
Ok I agree with this going out. I'll open a ticket for the lack of `serverUrl` being a limitation for document sharing between users.